scss/civicrm/common/_buttons.scss

```diff
@@ -6,7 +6,8 @@
 $button-border-radius: 2px;
 
 %btn-civi {
-  @extend .btn, %open-sans-font-family;
+  @extend .btn;
```

Comment:
As far as I remember we don't use `@extend`? This comment is related to all places where `extend` is left.

Reply:
`.btn` is a bootstrap class and here  the `btn-civi` needs the styles form bootstrap button. And since the bootstrap does not have a `mixin` for the button. Its necessary to use `extend` in this case.